docs/pages/routing/appearance.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Appearance elements
+description: Learn how to use a splash screen, fonts and images in your app that is using Expo Router.
+hideFromSearch: true
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+There are three major elements that you can use to customize the appearance of your app:
+
+- Splash screen
+- Fonts
+- Images
+
+## Splash screen
```

Comment:
I may deprecate this API, so maybe move it toward the bottom of the page

Reply:
Updated the ordering of sections as per your comment in this page.